modules/500-upmeter/monitoring/prometheus-rules/alerting.yaml

```
@@ -313,26 +364,33 @@
         plk_create_group_if_not_exists__d8_upmeter_resources_garbage: "D8UpmeterProbeGarbage,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
         plk_grouped_by__d8_upmeter_resources_garbage: "D8UpmeterProbeGarbage,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
         plk_labels_as_annotations: "pod"
-        summary: Garbage produced by scheduler probe is not being cleaned.
+        summary: Garbage Pods from the scheduler probe are not being cleaned up.
         description: |
-          Average probe pods count per upmeter-agent pod: {{ $value }}.
+          The average number of probe Pods per `upmeter-agent` Pod is {{ $value }}.
 
-          Upmeter agents should clean Pods produced by control-plane/scheduler probe. There should not be more
-          of these pods than master nodes (upmeter-agent is a DaemonSet with master nodeSelector). Also, they should be
-          deleted within seconds.
+          `upmeter-agent` should automatically clean up Pods created by the `control-plane/scheduler` probe.  
+          There should not be more of these Pods than there are master nodes (since `upmeter-agent` runs as a DaemonSet with a master `nodeSelector`), and they should be deleted within seconds.
 
-          This might be an indication of a problem with kube-apiserver. Or, possibly, the pods were left
-          by old upmeter-agent pods due to Upmeter update.
+          This may indicate:
 
-          1. Check upmeter-agent logs
+          - A problem with the `kube-apiserver`.
+          - Stale Pods left from old `upmeter-agent` Pods after an Upmeter update.
 
-          `kubectl -n d8-upmeter logs -l app=upmeter-agent --tail=-1 | jq -rR 'fromjson? | select(.group=="control-plane" and .probe == "scheduler") | [.time, .level, .msg] | @tsv'`
+          Troubleshooting steps:
 
-          2. Check that control plane is functional.
+          1. Check `upmeter-agent` logs:
 
-          3. Delete pods manually:
+             ```shell
+             kubectl -n d8-upmeter logs -l app=upmeter-agent --tail=-1 | jq -rR 'fromjson? | select(.group=="control-plane" and .probe == "scheduler") | [.time, .level, .msg] | @tsv'
+             ```
 
-          `kubectl -n d8-upmeter delete po -l upmeter-probe=scheduler`
+          2. Ensure the control plane is operating normally.
+
+          3. Delete stale Pods manually:
+
+             ```shell
+             kubectl -n d8-upmeter delete po -l upmeter-probe=scheduler
```

It is suggested to specify full commands everywhere, not aliases. 

```suggestion
             kubectl -n d8-upmeter delete pods -l upmeter-probe=scheduler
```